src/components/Api.js

```diff
@@ -0,0 +1,147 @@
+export class Api {
+  constructor({ baseUrl, headers }) {
+    this._url = baseUrl;
+    this._authorization = headers.authorization;
+  }
+
```

Comment:
* The close icon in `Confirmation popup` is not correctly positioned http://joxi.ru/DrlW6XVSde8e5A